internal/dependencymanager/dependencyinstaller.go

```diff
@@ -74,6 +76,11 @@ type dependencyManagerFlagsCollection struct {
 	skipAlias       bool `default:"false" flag:"skip-alias" info:"Skip prompting for an alias"`
```

Comment:
If we're not using the sconfig metadata, we should probably just remove it.  Otherwise this could be misleading for someone looking at the code later